server/graphql/v2/mutation/index.js

```diff
@@ -1,5 +1,6 @@
 import accountMutations from './AccountMutations';
 import { addFundsMutation } from './AddFundsMutations';
+import applicationMutations from './ApplicationMutations';
```

Comment:
We're doing `export const` in there so we probably need

```suggestion
import * as applicationMutations from './ApplicationMutations';
```

Reply:
But we'll need to rename the exports if we do that

Reply:
I moved back to the approach we had in other files.